inc/ticket.class.php

```diff
@@ -210,9 +210,30 @@ static function addHistoryOnAddGroup(CommonDBTM $item) {
 
       //add line in history table
       $history = new PluginEscaladeHistory();
+
+      $group_ticket      = new Group_Ticket();
+      $last_group_tickets = $group_ticket->find("tickets_id = $tickets_id AND type = 2", "id DESC", 1);
+
+      $previous_groups_id = 0;
+      $counter            = 0;
+
+      if (count($last_group_tickets) > 0) {
+         $last_group_ticket  = array_shift($last_group_tickets);
+         $previous_groups_id = $last_group_ticket['groups_id'];
+
```

Comment:
I do not know how the plusing works, but maybe you should skpi next lines if `$last_group_ticket['groups_id'] == $groups_id` as it does not looks like an escalation.